datafusion/core/src/datasource/listing/table.rs

```diff
@@ -1584,11 +1590,127 @@ mod tests {
         helper_test_append_new_files_to_table(
             FileType::CSV,
             FileCompressionType::UNCOMPRESSED,
+            None,
         )
         .await?;
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_insert_into_append_new_parquet_files_defaults() -> Result<()> {
+        helper_test_append_new_files_to_table(
+            FileType::PARQUET,
+            FileCompressionType::UNCOMPRESSED,
+            None,
+        )
+        .await?;
+        Ok(())
+    }
+
+    #[tokio::test]
+    async fn test_insert_into_append_new_parquet_files_session_overrides() -> Result<()> {
```

Comment:
We may want to add additional tests for writing parquet that fuzz these input settings and write out some larger parquet files so that the configs actually have some effect.